analysis/report_templates/default.html

```diff
@@ -232,7 +232,25 @@ <h5 class="center-align">Mann-Whitney U test</h4>
                 </li>
             </ul>
 
-
+            {% if coverage_report %}
+            <ul class="collapsible">
+                <li>
+                    <div class="collapsible-header">
+                        Coverage reports for each fuzzer on this benchmark
+                    </div>
+                    <div class="collapsible-body">
+                        <div class="row">
+                            {% for fuzzer in benchmark.fuzzers %}
+                            <div class="col">
+                                <a class="waves-effect waves-light btn" href="https://storage.googleapis.com/{{ experiment_filestore_name }}/{{ experiment.name }}/reports/coverage/{{ benchmark.name }}/{{ fuzzer }}/index.html">{{ fuzzer }}</a>
```

Comment:
would it make sense to make this whole path (https://storage.googleapis.com/{{ experiment_filestore_name }}) a variable (eg `reports_path`) instead of just `experiment_filestore_name`, in order to make this more flexible (eg support local paths)

Reply:
alternatively, we could use `experiment_filestore` from the experiement-config.yaml, and if it's a gs:// path then use https://storage.googleapis.com/, otherwise not.